extension-basics.md

```diff
@@ -38,12 +38,12 @@ Get how-tos, resources, and information to successfully build and ship your exte
 The Extension Workshop can help you develop extensions for Firefox and give your users simple, yet powerful ways to customize their browsing experience. You’ll find:
 
 - Overview of the Firefox extension features
-- Tools and processes for developing and testing
-- How to publish your extension on addons.mozilla.org or distribute it yourself.
-- How to manage your published extension
-- An enterprise guide for developing and using extensions
-- How to develop themes for Firefox
-- Firefox developer communities
+- [Tools and processes for developing and testing](https://extensionworkshop-dev.allizom.org/documentation/develop/)
+- [How to publish your extension on addons.mozilla.org or distribute it yourself](https://extensionworkshop-dev.allizom.org/documentation/publish/)
```

Comment:
You missed the period (`.`) here

Reply:
The period in list format looks misleading. Do I still need to add it?